src/cli/build.ts

```diff
@@ -180,9 +176,9 @@ export default function (program: RootCmd) {
 
       let zarf = "";
       if (opts.zarf === "chart") {
-        zarf = assets.zarfYamlChart(chartPath);
+        zarf = await assets.zarfYamlChart(chartPath);
```

Comment:
this should not be async

Reply:
Fixed these by making the functions in assetsDeployer.ts no be async.